### DISCUSSION TITLE
I would like to start a PERMIT COMMITTEE

### DISCUSSION THREAD

**Discussion by happiah-madson** [ArctosDB/arctos#8275] [Announcements]
I need a permit committee to exist so that we can get some permit things done.

People who I think may be interested include:
@campmlc @mkoo @dustymc 

Please add other names or collections because I know nearly no one!

Here is a when2meet poll for the month of November, because I would like to get this moving before the new year!

https://www.when2meet.com/?27414736-SsK6B

**Comment by campmlc** [comment]
I added some times. Ideally it could be at 10am MDT on alternate weeks from Object Tracking?

**Reply by mkoo** [reply]
I could do this

**Comment by mkoo** [comment]
Great idea! I am going to convert to a discussion to continue the conversation AND send an AWG email to get more ideas and people to join.  Hannah, you're leading, correct?! Thanks!

**Reply by mkoo** [reply]
email to listserv sent for recruitment

**Comment by happiah-madson** [comment]
Yep, I can lead!

**Comment by dustymc** [comment]
I added some times. Assuming the agenda (which would help me prioritize, I'll add a note to my hopeful-recipe) is about what I hope it will be, I suggest avoiding the word 'permit' (which might imply that this is limited to whatever someone might think the current permit node is). I'm not sure I have a better word, but my hope is that this, whatever it becomes, will result in one unified place for "usage guidance" (?? not quite correct, I still don't have a better term) data.

**Comment by happiah-madson** [comment]
For better or worse, I have created a project: https://github.com/orgs/ArctosDB/projects/205

I've called it "usage conditions" ? But also maybe, "material usage conditions" ?

In looking at the availability on the calendar, ya'll really don't have aligned time. @dustymc @mkoo I don't think there is a single time you can both be there and it feels important for you to both be there. Should I open up times in December?

**Comment by dustymc** [comment]
>material

I'd avoid even that - eg 'you can use this record's data but only if you ......' fits well within my idea of a 'permit.'

"usage conditions" is nice!

>availability

This seems pretty critical to me, and I think we've got some momentum from yesterday which I'd like not to lose, I think I can find a way to make one meeting and then we can re-evaluate if we need more?

And https://github.com/ArctosDB/arctos/issues/8241#issuecomment-2463091583 - maybe we've got an actionable start, I think I was hearing universal agreement on adding the pair of 'category' fields to permits - for internal only use at the moment - but not sure if the names/concepts are solidified yet. And acting on the comment below that re: encumbrances - possibly that needs a new dedicated Issue?? - would be a nice consolidation which would make adding this summary (I think that's the plan...) to various UI (eg loan pick) immediately functional/useful.

**Comment by happiah-madson** [comment]
> one meeting and then we can re-evaluate if we need more

Perfect. Friday November 15 between 12 and 2 pm EST (9 and 11 PST) seems like a good time for others who have responded to the poll. Can you swing that @dustymc ?

**Reply by dustymc** [reply]
Yes, I can arrange next Friday at 9, thanks!

**Reply by happiah-madson** [reply]
@mkoo I don't think I can add an event to the Arctos calendars. I'm I just being dense, or can you help?

**Comment by happiah-madson** [comment]
Reminder that we are meeting today to discuss permits (to be possibly known as Usage Conditions) TODAY, Friday November 15 at 9 am PT/12 pm ET!

I still can't add things to the Arctos Calendar, but I presume we can use the Arctos Zoom Room!

**Comment by dustymc** [comment]
Its lonely in https://berkeley.zoom.us/j/91591455018....

**Comment by campmlc** [comment]
I am interested in a new meeting for 2025. This month is crazy, but perhaps a week when we don't have an AWG or issues meeting, or object tracking.

**Comment by happiah-madson** [comment]
Just want to come here and say I added usage conditions to a permit! I can see them in the search accession page! I can see them in the permit summary page! I can see them on a record! <3 <3 <3 <3 <3 I look forward to the continued journey of improving this new & amazing feature! Thank you @dustymc !!!!!!!

**Comment by happiah-madson** [comment]
Permit/usage committee meeting now! link here: https://northeastern.zoom.us/j/97016687524?pwd=8kn25a9ATzvVhvmHb4GwxFnNrqpA7o.1
